test/gtest/ucs/test_bitmap.cc

```diff
@@ -352,6 +352,22 @@ class test_dynamic_bitmap : public ucs::test {
 
 protected:
     ucs_dynamic_bitmap_t m_bitmap;
+
+    static void set(ucs_dynamic_bitmap_t &bitmap,
+                    const std::initializer_list<size_t> &offsets)
+    {
+        for (auto offset : offsets) {
+            ucs_dynamic_bitmap_set(&bitmap, offset);
+        }
+    }
+
+    void expect_bits(const std::initializer_list<size_t> &offsets)
```

Comment:
IMO it would be better to have static function that takes bitmap by const reference, rather than using member `m_bitmap`. This way it's more reusable.
If you prefer to keep it as is, then this function may be const:
`void expect_bits(const std::initializer_list<size_t> &offsets) const`

Reply:
changed to const